src/main/starlark/builtins_bzl/common/cc/attrs.bzl

```diff
@@ -84,6 +84,23 @@ using the C/C++ compiler.
 </p>
 """,
     ),
+    "module_interfaces": attr.label_list(
+        allow_files = True,
+        doc = """
+The list of files are regarded as C++20 Modules Interface.
+
+<p>
+C++ Standard has no restriction about module interface file extension
+<ul>
+<li>Clang use cppm </li>
+<li>GCC can use any source file extension </li>
+<li>MSVC use ixx </li>
+</ul>
+</p>
+<p>For now usage is guarded by the flag
```

Comment:
nit: `For now usage` -> `The use`

Technical documentation should avoid using `for now` or other time constructs. Especially at Google :)